.github/workflows/ci.yml

```diff
@@ -38,7 +40,7 @@ jobs:
         username: nextstrain-bot
         password: ${{ secrets.GH_TOKEN_NEXTSTRAIN_BOT_MANAGE_PACKAGES }}
 
-    - run: ./devel/build -p linux/amd64 -r ghcr.io -t $TAG
+    - run: ./devel/build -p linux/amd64,linux/arm64 -r ghcr.io -t $TAG
```

Comment:
_non-blocking_

Can we build for the two platforms separately and in parallel in different matrix jobs?

Reply:
I believe it's possible, though would require manual pushing/pulling of intermediate images and stringing them together with `docker manifest` to produce a single multi-platform image ("the old way" as described in [this blog post](https://dev.to/cloudx/multi-arch-docker-images-the-easy-way-with-github-actions-4k54)).

This is a trade-off between implementation complexity and build time (which I assume is the main benefit from running in parallel). Currently, the single-platform amd64 build takes [~6 minutes](https://github.com/nextstrain/docker-base/actions/runs/3700380538/jobs/6268749382) while the amd64+arm64 build takes [>1 hour](https://github.com/nextstrain/docker-base/actions/runs/3632781633/jobs/6129028596) on a single job. That means running in parallel would in theory only result in a ~6 minute decrease, which I don't think is enough of an improvement to warrant the increased complexity.

I think BuildKit itself takes advantage of the number of cores on the runner. So another option for parallel platform builds would be to use a runner with more cores.

Reply:
Sounds good. Thanks for explaining!

> …which I don't think is enough of an improvement to warrant the increased complexity.

Agreed!

> So another option for parallel platform builds would be to use a runner with more cores.

Good to keep in mind if (when) the build times get us down.